### ISSUE TITLE
Asset: dcat:landingpage range

### ISSUE THREAD

**Issue by sandervd** [SEMICeu/ADMS-AP#15]
The range of the property is now foaf:Document, which suggests that it should point to an instance of foaf:Document.
I know that the disjoint argument will probably follow here, but this openness can be problematic for implementation.
I find it confusing to set the range to a class instance, and then in the usage note specify that it should be any URI.

**Comment by sandervd** [comment]
Same for Asset repository foaf:homepage

**Comment by bertvannuffelen** [comment]
The property https://www.w3.org/TR/vocab-dcat-2/#Property:resource_landing_page states that the range. It is part of the definition. So you get it whether you like it or not. ;-)

This is part of the reuse challenge and how vocabularies are being defined. But if in the defining document of the property the range is included, it must be considered an integral part of the definition. We cannot shop what we like to reuse.

However we can tackle this topic in the data exchange process with a closure statement. Namely we could trust that the values for this property are members of foaf:Document. And insert this statement while parsing/processing the received data.

Similar discussions are in the DCAT-AP issues, because above arguments lead to the usage of different levels of severity in SHACL files or other modularisation of the constraints over different SHACL files.

**Comment by sandervd** [comment]
Yup, I think this can partially be solved by improving how we show things in the documentation.

Then, how open/closed/restricted the SHACL representation should be...
I know it is not according to semweb traditions to close and restrict, but it can be very helpful to actually be able to validate a document in a meaningful way.
I know this will raise a discussion (at least I hope it does), but in my opinion it is perfectly acceptable to close the world in an application profile. (/me hides)
